src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -3737,7 +3737,7 @@ public Long xlen(final String key) {
    * {@inheritDoc}
    */
   @Override
-  public long xack(final String key, final String group, final StreamEntryID... ids) {
+  public Long xack(final String key, final String group, final StreamEntryID... ids) {
```

Comment:
I think it can cause some backward issues

Reply:
That's why I've added this in major release. And it's necessary to maintain common/unified interface.